phpunit.xml.dist

```diff
@@ -10,6 +10,7 @@
     <php>
         <ini name="error_reporting" value="-1" />
         <server name="KERNEL_DIR" value="app/" />
+        <env name="DATABASE_URL" value="sqlite://%kernel.project_dir%/var/data/blog_test.sqlite"/>
```

Comment:
@dmaicher you are right. I fixed that in 464e786 but I think Christophe is right and container params don't work here.